packages/@o3r/create/src/index.ts

```diff
@@ -265,6 +266,21 @@ const prepareWorkspace = (relativeDirectory = '.', projectPackageManager = 'npm'
     }
   }
 
+  if (argv.minimal) {
+    const editorconfigPath = resolve(cwd, '.editorconfig');
```

Comment:
Why is it implemented in this file instead of be part of `core`?
To remember, this file should contain exclusively what it mandatory to run Ng generation and the add of Otter core.
If it is not the case of these file updates you may need to move them in core.

I also think that in any case, as the generators require to be have the `lf` constrain, the core schematics should do this update (as well if it is mandatory here).

Reply:
For me it's part of the file for the monorepo configuration I would expect to have it defined directly when I create the monorepo 
I don't expect that it changes if someone run `ng add @o3r/core` on an exisiting monorepo

Reply:
What do you think about that @AmadeusITGroup/otter_contributors  ?

Reply:
I also think this should only be generated at `create` time, not on `ng add`

Reply:
Same as Florian and Matthieu. I would not mess with an existing project. 
It makes sense to keep it aligned with the current project and in this case the file is created during the create process.

Reply:
But this is not at all the purpose of the create script.
If it is a mandatory for our generators then as soon as we add Otter to a project we may need this change.
If it is not, but part of the Otter guidelines, then it should be bring by a `ng add` (even if it is done only once).

Please remember that the create script is not mandatory to make the other installation work, should remain as simple as possible and should not be the one applying guidelines (or setup default value)